src/components/pages/GenerationPage/GenerationInput/AddressInput.js

```diff
@@ -26,7 +26,8 @@ const TextInput = styled(Input)`
 `;
 
 export default function AddressInput({ address, setAddress, disabled, errors, onClearError }) {
-  const { address1, address2, city, state, zipCode } = address;
+  const { address1, address2, city, state, zipcode } = address;
+  const stateObj = typeof state === 'string' ? { value: state, label: state } : state;
```

Comment:
Was the type of the `state` data inconsistent here? If so, I'd like to make it consistent if it's easy enough.